pkg/commands/flagsets.go

```diff
@@ -42,18 +42,18 @@ func setupFormattersFlagSet(v *viper.Viper, fs *pflag.FlagSet) {
 }
 
 func setupRunFlagSet(v *viper.Viper, fs *pflag.FlagSet) {
-	internal.AddFlagAndBindP(v, fs, fs.IntP, "concurrency", "j", "run.concurrency", getDefaultConcurrency(),
-		color.GreenString("Number of CPUs to use (Default: number of logical CPUs)"))
+	internal.AddFlagAndBindP(v, fs, fs.IntP, "concurrency", "j", "run.concurrency", 0,
+		color.GreenString("Number of CPUs to use (Default: Automatically set to match Linux container CPU quota,"+
+			" fall backs on number of logical CPUs)"))
```

Comment:
Description can be the same as in the `.golangci.next.reference.yml`:
```suggestion
		color.GreenString("Number of CPUs to use (Default: Automatically set to match Linux container CPU quota"+
			" and fall back to the number of logical CPUs in the machine)"))
```